blazor/treeview/integration.md

```diff
@@ -0,0 +1,1133 @@
+                            <p class="main-menu-header">TABLE OF CONTENTS</p>
+                        </div>
+                        <div>
+                            <SfTreeView CssClass="main-treeview" @bind-SelectedNodes="@selectedNodes" @ref="tree" ExpandOn="@Expand" TValue="TreeData">
+                                <TreeViewFieldsSettings Id="NodeId" NavigateUrl="NavigateUrl" Text="NodeText" IconCss="IconCss" DataSource="Treedata" HasChildren="HasChild" ParentID="Pid">
+                                </TreeViewFieldsSettings>
+                                <TreeViewEvents TValue="TreeData" ></TreeViewEvents>
+                            </SfTreeView>
+                        </div>
+                    </div>
+                </ChildContent>
+            </SfSidebar>
+            <!-- end of sidebar element -->
+            @*main-content declaration*@
+            <div class="main-content" id="main-text" onclick="click()">
+                <div class="sidebar-content">
+                    <div class="content">
+                        @Body
+                    </div>
+                </div>
+                <!--end of main content declaration -->
+            </div>
+        </div>
+    </div>
+</body>
+@code {
+    public string[] selectedNodes = new string[] { "01" };
+    // Specifies the value of TreeView component ExpanOn property.
+    public ExpandAction Expand = ExpandAction.Click;
+    // Specify the value of Sidebar component state. It indicates whether the sidebar component is in an open/close state.
+    public bool SidebarToggle = false;
+    // Specifies the value of Sidebar HTMLAttribute property.
+    Dictionary<string, object> HtmlAttribute = new Dictionary<string, object>()
+{
+        {"class", "sidebar-treeview" }
+    };
+    // Specifies the event handler for Clicked event in Toolbar component.
+    public void Toggle(ClickEventArgs args)
+    {
+        if (args.Item.TooltipText == "Menu")
+        {
+            SidebarToggle = !SidebarToggle;
+        }
+    }
+    SfTreeView<TreeData> tree;
+    public class TreeData
+    {
+        public string NodeId { get; set; }
+        public string NodeText { get; set; }
+        public string IconCss { get; set; }
+        public bool HasChild { get; set; }
+        public string Pid { get; set; }
+        public string NavigateUrl { get; set; }
+    }
+    private List<TreeData> Treedata = new List<TreeData>();
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        Treedata.Add(new TreeData { NodeId = "01", NodeText = "Installation", IconCss = "icon-microchip icon", NavigateUrl = "/" });
+        Treedata.Add(new TreeData { NodeId = "02", NodeText = "Deployment", IconCss = "icon-thumbs-up-alt icon", NavigateUrl = "/counter" });
+        Treedata.Add(new TreeData { NodeId = "03", NodeText = "Quick Start", IconCss = "icon-docs icon", NavigateUrl = "/fetchData" });
+    }
+}
+
+<style>

```

Comment:
strict bootstrap 5 theme in CSS (instead of all themes)

Reply:
Done.